packages/flutter/test/material/checkbox_test.dart

```diff
@@ -1446,6 +1447,57 @@ void main() {
     await tester.pumpAndSettle();
     expectBorder();
   });
+
+  testWidgets('disabled checkbox shows tooltip', (WidgetTester tester) async {
+    const String longPressTooltip = 'long press tooltip';
+    const String tapTooltip = 'tap tooltip';
+    await tester.pumpWidget(
+        const MaterialApp(
+          home: Material(
+            child: Tooltip(
+              message: 'long press tooltip',
+              child: Checkbox(value: true, onChanged: null),
+            ),
+          ),
+        )
+    );
+
+    // Default tooltip shows up after long pressed.
+    final Finder tooltip0 = find.byType(Tooltip);
+    expect(find.text(longPressTooltip), findsNothing);
+
+    await tester.tap(tooltip0);
+    await tester.pump(const Duration(milliseconds: 10));
+    expect(find.text(longPressTooltip), findsNothing);
+
+    final TestGesture gestureLongPress = await tester.startGesture(tester.getCenter(tooltip0));
+    await tester.pump();
+    await tester.pump(kLongPressTimeout);
+    await gestureLongPress.up();
+    await tester.pump();
+
+    expect(find.text(longPressTooltip), findsOneWidget);
+
+    // Tooltip shows up after tapping when set triggerMode to TooltipTriggerMode.longPress.
+    await tester.pumpWidget(
+        const MaterialApp(
+          home: Material(
+            child: Tooltip(
+              triggerMode: TooltipTriggerMode.tap,
+              message: tapTooltip,
+              child: Checkbox(value: true, onChanged: null),
+            ),
+          ),
+        )
```

Comment:
nit: remove extra indentation
(same for the previous call to `pumpWidget` in this file and in the other test files)